adam/experiment/log_experiment.py

```diff
@@ -51,10 +51,10 @@ def log_experiment_entry_point(params: Parameters) -> None:
             name=experiment_name,
             training_stages=training_instance_groups,
             learner_factory=learner_factory_from_params(params, graph_logger),
-            pre_example_training_observers=[logger.pre_observer()],
-            post_example_training_observers=[logger.post_observer()],
+            pre_example_training_observers=[logger.pre_observers()],
```

Comment:
Why change the name to plural but still only return one thing?

Reply:
Good catch. That was an edit that I went back and forth on.